seed-db.ts

```diff
@@ -1478,6 +1478,68 @@ void (async function setupDevDB() {
         },
     });
 
+    // STUDENT PARTICIPATION STREAK
+    await prisma.achievement_template.create({
+        data: {
+            name: 'Anwesenheits',
+            metrics: ['student_participation_streak'],
+            templateFor: achievement_template_for_enum.Global,
+            group: 'student_participation',
+            groupOrder: 1,
+            stepName: '',
+            type: achievement_type_enum.STREAK,
+            subtitle: 'Du warst bei {{recordValue}} Termin(en) in Folge anwesend!',
+            description:
+                'Du warst bei {{recordValue}} Termin(en) in Folge dabei! Behalte diesen großartigen Trend bei und steigere ihn noch weiter. Jedes Mal, wenn du zu einem Termin erscheinst, steigt deine Teilnahme-Serie. Deine konstante Ausdauer könnte dich bis zum Teilnahme-Marathon führen. Mach weiter so, du bist auf dem besten Weg zum Erfolg!',
```

Comment:
This would not work. We do not have a WYSIWYG field here that could detect line breaks. Also Termine is not the singular, it would be "1 Termin" and "2 Termin(en)". I know the files show it different, but that would be wrong grammar.